pandas/io/json/json.py

```diff
@@ -263,6 +266,16 @@ def read_json(path_or_buf=None, orient=None, typ='frame', dtype=True,
 
         .. versionadded:: 0.19.0
 
+    chunksize: integer, default None
+        If `lines=True`, how many lines to read into memory at a time.
+        If this is None, the file will be read into memory all at once.
+        Passing a chunksize helps with memory usage, but is slower.
+        Also note this is different from the `chunksize` parameter in
+            `read_csv`, which returns a FileTextReader.
```

Comment:
Shouldn't our goal be to have it with a similar behaviour?

(formatting comment: there is some extra indentation on this line that is not needed

Reply:
need to update these doc-strings